connext_cmake_module/CMakeLists.txt

```diff
@@ -1,9 +1,21 @@
 cmake_minimum_required(VERSION 2.8.3)
 
-project(connext_cmake_module NONE)
+project(connext_cmake_module)
```

Comment:
It looks like `FindConnext.cmake` should make sure to perform the necessary compile checks in order to be used from this with `NONE` being set.